tgui/packages/tgui/interfaces/AiRestorer.tsx

```diff
@@ -84,6 +84,14 @@ export const AiRestorerContent = (props, context) => {
             mt={1}
             onClick={() => act('PRG_beginReconstruction')}
           />
+          <Button
```

Comment:
`{/* SKYRAT ADDITION - Malf Remover */}`

Above this line, and the same with an end comment after the button, just to make it easier to figure out if this is an us thing in the future.

Reply:
done thank u